public/swagger/index.tsx

```diff
@@ -28,9 +28,9 @@ import { Page } from './SwaggerPage';
 const tt = window.trustedTypes;
 if (tt?.createPolicy) {
   tt.createPolicy('default', {
-    createHTML: (string, sink) => DOMPurify.sanitize(string, { RETURN_TRUSTED_TYPE: true }) as unknown as string,
-    createScriptURL: (url, sink) => textUtil.sanitizeUrl(url) as unknown as string,
-    createScript: (script, sink) => script,
+    createHTML: (string) => DOMPurify.sanitize(string, { RETURN_TRUSTED_TYPE: true }) as unknown as string,
+    createScriptURL: (url) => textUtil.sanitizeUrl(url) as unknown as string,
+    createScript: () => "",
```

Comment:
Not a bad idea. Does this prohibit sending requests from the Swagger UI though?

Reply:
I tested all the UI features I use (sending requests, etc) and it all seems good

Reply:
That is controlled by the `connect-src` directive, which is not set. The goal of this PR is to protect against XSS. The trusted types policy will just interfere with using "unsafe" DOM API, and should not disturb any requests from/to Swagger UI.